src/DotLiquid/StandardFilters.cs

```diff
@@ -65,13 +65,34 @@ public static int Size(object input)
             return 0;
         }
 
+        /// <summary>
+        /// Returns a substring of one character or series of array items beginning at the index specified by the first argument.
+        /// </summary>
+        /// <param name="input">The input to be sliced</param>
+        /// <param name="start">zero-based start position of string or array, negative values count back from the end of the string/array.</param>
+        /// <param name="len">An optional second argument specifies the length of the substring or number of array items to be returned</param>
+        public static object Slice(object input, int start, int len = 1)
+        {
+            if (input is IEnumerable<string> enumerableInput)

```

Comment:
not clear why this needs to be limited to ienumerable of type string. All ienumerable, except for strings (which are ienumerable<char>)  should be supported if the spec doesn't say otherwise.

Reply:
the ruby code says: 
if input.is_a?(Array)
I'm not a Ruby programmer, but I'm sure this can be tested further to validate. From what I see, any array is supported.

Reply:
I'm not a ruby programmer either, so thanks for checking.